docs/cookbook/stripe.rst

```diff
@@ -50,7 +50,12 @@ Having these parameters defined you can configure the gateway inside the ``app/c
 Add a payment method with the Stripe Credit card gateway in the Admin Panel
```

Comment:
The configuration of Stripe stays unchanged? I mean the `Configure the gateway` chapter.